scripts/split-paired-reads.py

```diff
@@ -72,11 +73,13 @@ def get_parser():
                         'directory if necessary')
 
     parser.add_argument('-1', '--output-first', metavar='output_first',
+                        type=argparse.FileType('w'),
                         default=None, help='Output "left" reads to this '
-                        'file', type=argparse.FileType('w'))
```

Comment:
You have the noisiest diffs! Half of these changes make no difference; what's up with that? :-)